cds/modules/records/templates/cds_records/record_detail.html

```diff
@@ -14,10 +14,19 @@
 {%- endblock page_footer %}
 
 {%- block page_body %}
-  <div id="cds-record">
-    <cds-record-view
-        template="{{ url_for('static', filename='templates/cds_records/video/detail.html') }}"
-        record='{{ record | tojson }}'
-    ></cds-record-view>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <img data-toggle="modal" data-target="[data-modal='{{ record.doi }}']"
+          src="{{ url_for('invenio_formatter_badges.badge', title='DOI', value=record.doi, ext='svg') }}" alt="{{ record.doi }}">
+        </img>
+        <div id="cds-record">
+          <cds-record-view
+            template="{{ url_for('static', filename='templates/cds_records/detail.default.html') }}"
```

Comment:
We already have the badge on 

- [detail.default.html](https://github.com/hachreak/cds/blob/34fea8943f2d7c390b112c8f3ba9eea33a1b3b0c/cds/modules/records/static/templates/cds_records/detail.default.html#L53)
- [video/detail](https://github.com/hachreak/cds/blob/34fea8943f2d7c390b112c8f3ba9eea33a1b3b0c/cds/modules/records/static/templates/cds_records/video/detail.html#L28)

Could you please update it on the jinja part, create a new ``static/templates/cds_records/doi.html``

```html
 <a class="get-badge" ng-if="identifier.source_of_number_or_code == 'DOI'" ng-href="http://dx.doi.org/{{identifier.standard_number_or_code}}" target="_blank">
   <img ng-src="/badge/doi/{{record.doi}}.svg" alt="{{ record.doi }}">
</a>
```

And replace the existing doi html snippet with

```HTML
 <ng-include src="'static/templates/cds_records/doi.html'"></ng-include>
```

In this way, for every new template we'll just include the ``doi`` and voila!